captum/attr/_core/noise_tunnel.py

```diff
@@ -205,7 +205,7 @@ def compute_expected_attribution_and_sq(attribution):
 
             attribution = attribution.view(attribution_shape)
             expected_attribution = attribution.mean(dim=1, keepdim=False)
-            expected_attribution_sq = torch.mean(attribution ** 2, dim=1, keepdim=False)
+            expected_attribution_sq = torch.mean(attribution**2, dim=1, keepdim=False)
```

Comment:
@ProGamerGov, this PR has also formatted the files in the non optim related code. Perhaps you can bring this back. Also, maybe you have a different version of code formatter that's why it changed the formatting ?

Reply:
@NarineK The black tests fail currently because the optim-wip branch's version of black was never fixed to a specific version. I've reverted the changes.